src/main/java/com/portatlas/FileReader.java

```diff
@@ -0,0 +1,29 @@
+package com.portatlas;
+
+public class FileReader {
+
+    public static String getMediaType(String resource) {
+        String fileExtension = getFileExtension(resource);
+
+        switch(fileExtension) {
+            case "txt" :
+                return "text/plain";
+            case "png" :
+                return "image/png";
+            case "jpeg" :
+                return "image/jpeg";
+            case "gif" :
+                return "image/gif";
+            default:
+                return "application/octet-stream";
+        }
```

Comment:
Would it be more readable to build a hashmap where the key is the file extension?